src/Microsoft.DotNet.Interactive/Microsoft.DotNet.Interactive.csproj

```diff
@@ -12,6 +12,19 @@
     <PackageDescription Condition="'$(PackageDescription)'==''">Core types for building applications providing interactive programming for .NET.</PackageDescription>
     <PackageTags Condition="'$(PackageTags)'==''">polyglot notebook dotnet interactive</PackageTags>
   </PropertyGroup>
+  
+  <ItemGroup Label="Resources">
+    <!-- JavaScript kernel -->
+    <EmbeddedResource Include="..\polyglot-notebooks\lib\polyglot-notebooks.js" />
```

Comment:
Embedding this in this library seems odd. What pulled this over here?

Reply:
Ah this is needed on the VS side and was originally part of `Microsoft.DotNet.Interactive.VisualStudio.dll`. I moved it here as part of deleting `Microsoft.DotNet.Interactive.VisualStudio.csproj`. I just remembered that we had said we could put this resource in the `BrowserKernels` package, multi-target that to support `netstandard2.0` and fetch it from there instead. Let me look into that...